src/styles/card.scss

```
@@ -0,0 +1,115 @@
+@import "../../layout_stars/src/style.css";
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.card {
+  display: flex;
+  padding: 0 16px;
+  width: 200px;
+  flex-direction: column;
+  justify-content: center;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    margin: 32px 3px;
+  }
+
+  &__pic {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin-top: 5px;
+  }
+  
+  &__description {
+    line-height: 18px;
+    color: #060b35;
+  }
+  
+  &__product_code {
+    margin-top: 4px;
+  }
+  
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+  }
+
+  &__reviews {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 16px;
+    color: #060b35;
+  }
+  
+  &__rating {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-top: 16px;
+  }
+
+  &__stars {
+    display: flex;
+    flex-direction: row;
+  }
+
+  &__prices {
+    margin-top: 24px;
+    line-height: 18px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+  
+  &__price {
+    font-weight: 400;
+    font-size: 12px;
+    color: #616070;
+  }
+  
+  &__amount {
+    font-weight: 700;
+    font-size: 16px;
+  }
+  
+  &__buy {
+    margin: 16px 0;
+    text-align: center;
+    line-height: 40px;
+    box-sizing: border-box;
+  }
+  
+  &__buy-link {
+    text-decoration: none;
+    color: #fff;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    padding: 10px 68px;

```

- should be 11px, exclude 1-px border twice
- just make it spread all over given space by using proper `width` value
```suggestion
    padding: 11px 0;
```